src/w3d/renderer/camera.cpp

```diff
@@ -371,7 +371,7 @@ void CameraClass::Get_D3D_Projection_Matrix(Matrix4 *set_tm)
 
     *set_tm = m_projectionTransform;
 
-    float oozdiff = 1.0 / (m_zFar - m_zNear);
+    float oozdiff = double(1.0) / (m_zFar - m_zNear);
```

Comment:
what are you trying to achieve here. double cast to double???

Reply:
The purpose here is to mark this as double explicitly, so there is no question whether or not this is float. For compile this makes no difference.

Reply:
For reading it makes no difference either. Remove

Reply:
It does. When looking at it as is, one may think it should be a float (which it would be if we wrote it).

Reply:
So you are suggesting every time we use a double we add `double()`?

Reply:
It does help to signal "Hey, this is a double, although you would expect a float here."

If a double is used in double context, it would be not necessary to highlight explicit double. But here double is used in float context, so marking explicit double helps understanding that this is meant to be double. Alternatively we can add a comment instead.

Reply:
We already have excessive amounts of casts.

This doesn't make any difference to the sum in any context i see no reason in doing this. 1.0f and 1.0 are exactly the same. The only difference is which load instruction is used and wasting slightly more storage with a double.

We set the x87 flag to 32bit float calculations so all doubles are floats when calculating (except we don't actually use x87 maths so the flag does nothing for now so actually it's the reverse all floating point calculations are performed at long double precision).

Reply:
Replaced explicit double keyword with comment.